msi/createmsi.py

```diff
@@ -17,6 +17,7 @@
 import sys, os, subprocess, shutil, uuid
 from glob import glob
 import platform
+import shutil
```

Comment:
__[Flake8]__

redefinition of unused 'shutil' from line 17


<!-- Comment by [SideCI](https://sideci.com) -->